src/plugins/intel_gpu/src/plugin/program_builder.cpp

```diff
@@ -318,6 +293,106 @@ ProgramBuilder::ProgramBuilder(cldnn::engine& engine, const ExecutionConfig& con
         m_networkOutputs = *outputs;
 }
 
+ProgramBuilder::ProgramBuilder(const std::shared_ptr<ov::Model>& model, cldnn::engine& engine,
+                                const ExecutionConfig& config,
+                                std::shared_ptr<ov::threading::IStreamsExecutor> task_executor)
+    : m_curBatch(-1)
+    , m_config(config)
+    , m_engine(engine)
+    , queryMode(false)
+    , m_task_executor(task_executor) {
+    if (m_task_executor == nullptr)
+        m_task_executor = cldnn::program::make_task_executor(m_config);
+
+    std::shared_ptr<ov::Model> ngraph_function(model);
```

Comment:
minor comment but could you change the name of ngraph_ ? because it is deprecated naming. How about just ov_model ? 

BTW can't we just use model as is ?

Reply:
Fix codes to use model instead of ngraph_function. thanks.